src/index.ts

```diff
@@ -7,6 +7,8 @@
 
 import uploadFileFeature from './features/upload-file/upload-file.feature'
 
-export default uploadFileFeature
+export * from './features/upload-file/upload-file.feature'
```

Comment:
I don't think these `export *`s are working, and would love any hints as to why. I ended up needing to import `registerComponents` directly from `upload-file.feature`